app/models/github_fetcher/user.rb

```diff
@@ -0,0 +1,27 @@
+module GithubFetcher
+  class User
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def json
```

Comment:
I like this approach, though it means that the caller has to know the details of the GitHub api path, which feels a bit odd. But perhaps I'm thinking about it wrong.